### PR TITLE
TraceTarget - Rename RawWrite to ForceTraceWriteLine with default = true

### PR DESCRIPTION
**Pull request by 304NotModified**
Fixes https://github.com/NLog/NLog/issues/3744

### CODE REVIEW

File: src/NLog/Targets/TraceTarget.cs [hunk 1]
```diff
@@ -66,8 +66,8 @@ public sealed class TraceTarget : TargetWithLayout
         /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
         /// </summary>
         /// <docgen category='Output Options' order='100' />
-        [DefaultValue(false)]
-        public bool RawWrite { get; set; }
+        [DefaultValue(true)]
+        public bool ForceTraceWriteLine { get; set; } = true;
```
Comment (snakefoot) [inline src/NLog/Targets/TraceTarget.cs:70]: Think you need to keep `RawWrite` around as obsolete (Property that just assigns `ForceTraceWriteLine`)

**Comment by 304NotModified** [comment]
Sonarqube broken...

image

File: src/NLog/Targets/TraceTarget.cs (hunk 1 above)
Comment (304NotModified) [reply src/NLog/Targets/TraceTarget.cs:83]: Maybe just keep the name RawWrite then.

Comment (snakefoot) [reply src/NLog/Targets/TraceTarget.cs:83]: Like how `ForceTraceWriteLine` explains what is going to happen. And when making the breaking change of change the default-behavior (to use Trace WriteLine by default). Then might as well do it nicely.

**Comment by snakefoot** [comment]
Starting to reconsider changing the default, since Azure Log Streaming actually listens to the LogLevel:

https://www.thebestcsharpprogrammerintheworld.com/2018/08/23/system-diagnostics-trace-application-logging-log-stream-on-azure-app-service-function-app/

Changing the default will actually affect applications depending on Azure Log Streaming. Maybe the setting should just be left alone, since better to have some output (and fixing it), than no output because everything suddenly changes to LogLevel-Verbose (Instead of Info, Warning, Error).

**Comment by 304NotModified** [comment]
yes, i'm fine without changing the default

**Pull request closed without merging**